examples/example-gauth/README.md

```diff
@@ -44,12 +44,8 @@ gcloud pubsub topics create Topic1
 [service account](https://cloud.google.com/docs/authentication/#service_accounts) in order to access
 Pub/Sub via gRPC APIs as described [here](https://cloud.google.com/iam/docs/creating-managing-service-accounts).
 Assign the [role](https://cloud.google.com/iam/docs/granting-roles-to-service-accounts) `Project -> Owner`
-and for Key type select JSON. Once you click `Create`, a JSON file containing your key is downloaded to
```

Comment:
Downloading the Json is still an option, we should not remove this step. Instead specify the same information that was suggested to add in the comment - they can point GOOGLE_APPLICATION_CREDENTIALS env variable to the json file location, but also that it is unnecessary to do so on Google platforms (Google App Engine / Google Cloud Shell / Google Compute Engine). This info you can see here https://github.com/googleapis/google-auth-library-java/blob/afe4ff9f1f3c406c72ecb66fba50d7463f620467/oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java#L123

Reply:
Fixed